pkg/test/framework/components/echo/config.go

```diff
@@ -84,6 +84,15 @@ type Config struct {
 	// If enabled, echo will be deployed as a "VM". This means it will run Envoy in the same pod as echo,
 	// disable sidecar injection, etc.
 	DeployAsVM bool
+
+	// The hub to be used to pull the image for the VM. `DeployAsVM` must be enabled.
+	VMHub string
```

Comment:
We already define HUB/TAG at the overall test scope, is there a reason we need to add another one here?

Reply:
Ah I think I see why. See my comment below

Reply:
Updated so that it only has one extra field: VMImage